sections/04-open-source.qmd

```diff
@@ -135,6 +141,43 @@ r |> filter(package == "shiny") |> growth_plot() + ggtitle("Shiny for R")
 
 {{< pagebreak >}}
 
+::: {.page-banner image_paths='("assets/images/hexes/ellmer.svg", "assets/images/hexes/chatlas.png", "assets/images/hexes/ragnar.png", "assets/images/hexes/gander-lores.png", "assets/images/hexes/mall.png")'}
+
+## AI and LLM Enablement
+
+::: {.cols-2}
+
+LLMs (large language models) are changing how data scientists work. 
+Posit's Open Source teams are building tools to help data scientists responsibly
+use LLMs in their analysis, leveraging them to help during development, 
+and incorporating LLM capabilities in the solutions they provide others.
+
+### Selection of packages for enabling LLMs in data science applications {.unlisted}
+
+- [Ellmer](https://ellmer.tidyverse.org/) makes it easy to use large language models (LLM) from R. It supports a  variety of LLM providers and implements a rich set of features including streaming outputs, tool/function calling and structured data extraction.
+- [Chatlas](https://posit.co/blog/announcing-chatlas/) is a flexible Python interface to many LLM providers (playing a similar role to ellmer). It supports tool use, function calling, and streaming responses.
+- [Ragnar](https://ragnar.tidyverse.org) brings Retrieval-Augmented Generation (RAG) to R. Helps users index their own data and get LLM responses with grounded answers.
+- [Querychat](https://github.com/posit-dev/querychat) adds an SQL-powered LLM to Shiny apps. It lets users explore data with natural language. Querychat works in both R and Python.
```

Comment:
Having got to here and heard "R and Python" a number of times, it makes me wonder if we should call out earlier (maybe in the introduction to the Open Source section), our focus on bringing out both R and Python versions of new packages.

Reply:
I would like to bring up something to that effect.  Where appropriate, we bring both.